spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -34,19 +42,19 @@ describe('GOVUK.analyticsGa4.linkTracker', function () {
       links = document.createElement('div')
       links.innerHTML =
           '<div class="fully-structured-external-links">' +
-              '<a href="http://www.nationalarchives.gov.uk/1" link_domain="http://www.nationalarchives.gov.uk"> National Archives </a>' +
-              '<a href="https://www.nationalarchives.gov.uk/2" link_domain="https://www.nationalarchives.gov.uk"></a>' +
-              '<a href="https://www.nationalarchives.gov.uk/3.pdf" link_domain="https://www.nationalarchives.gov.uk">National Archives PDF</a>' +
+              '<a href="http://www.nationalarchives.gov.uk/1" link_domain="http://www.nationalarchives.gov.uk" path="/1"> National Archives </a>' +
```

Comment:
Might help to have a comment explaining what the non-valid `path` attribute is for.